src/helpers/__mocks__/filters.ts

```diff
@@ -1,4 +1,4 @@
-const filters = [];
+const filters: never[] = [];
```

Comment:
This, and similar changes, are preferably changed to:

```suggestion
const filters: Iterable<never> = [];
```